packages/paste-core/components/badge/src/hooks.tsx

```diff
@@ -1,11 +1,9 @@
 import * as React from 'react';
 import {useUIDSeed} from '@twilio-paste/uid-library';
 
-import type {BadgeChildren} from './types';
-
 const DEFAULT_ICON_SIZE = 'sizeIcon10';
 
-export const useResizeChildIcons = (children: BadgeChildren): BadgeChildren => {
+export const useResizeChildIcons = (children: React.ReactNode): React.ReactNode => {
```

Comment:
Should this potentially be [`NonNullable<React.ReactNode>`](https://www.typescriptlang.org/docs/handbook/utility-types.html#nonnullabletype)?

Reply:
totally! will update.